packages/cli/src/cmds/open/serveAndOpenAppMap.ts

```diff
@@ -13,7 +13,7 @@ export default async function serveAndOpenAppMap(appMapFile: string) {
   const builtDir = join(__dirname, '../../../built');
   const baseDir = (await exists(builtDir))
     ? builtDir
-    : join(__dirname, '../../..');
+    : join(__dirname, '..', '..');
```

Comment:
I think this was an artifact left over from when the `built` directory contained `src`. The directory structure has since flattened by one level (i.e., the compiled contents of `src` are now directly added to `built`). When run via `npx`, the former directory is used (`'../../../built'`).